### DISCUSSION TITLE
建议提前转移支付宝，微信，银行卡资金，避免被冻结。

### DISCUSSION THREAD

**Discussion by MarquessZhonggua** [WeNeedHome/SummaryOfLoanSuspension#208] [Ideas]
RT

**Comment by alexwjj** [comment]
买黄金储备吧，到时候支付宝微信给你一封，也没办法

**Comment by lfrledh** [comment]
离婚

把债务分离开

把资产转移到安全的人名下

**Comment by Leon0312** [comment]
现在老百姓能存放的资金，哪个不在监管范围？ 虚拟币？

**Comment by CaiJiJi** [comment]
USDC BUSD DAI

**Reply by feather812002** [reply]
找个亲戚朋友的国外地址开个wise的账户，可以收美元欧元，
国内人民币转成稳定币(USDC,USDP,BUSD)汇款到国外交易所，然后就可以随意转成欧元，美元英镑了。
唯一一点就是交易所和wise都要做KYC， wise还好一点， 交易所很多都限制国内用户了，所以必须要国外地址！

**Reply by CaiJiJi** [reply]
护照能开挺多国外银行
国外地址可以在 https://www.anytimemailbox.com/ 购买 
交易所可以用FTX支持大部分欧洲/澳洲/东欧 银行出入金 美国/香港的话有些限制了交易所资金
也可以走合规的KraKen

**Comment by ppwq2022** [comment]
把钱给我，我给你保管，每年付点保管费就行

**Comment by white-cat** [comment]
去一下银行，把钱取出来，用现金吧！

**Comment by hq001a** [comment]
笑死，楼主让家人把钱取出来了没
